node-manager/src/utils.rs

```diff
@@ -8,3 +10,35 @@ pub fn set_panic_hook() {
     #[cfg(feature = "console_error_panic_hook")]
     console_error_panic_hook::set_once();
 }
+
+pub fn to_compressed_pubkey(hex: &str) -> Option<PublicKey> {
+    let data = match to_vec(&hex[0..33 * 2]) {
+        Some(bytes) => bytes,
+        None => return None,
+    };
+    match PublicKey::from_slice(&data) {
+        Ok(pk) => Some(pk),
+        Err(_) => None,
+    }
+}
+
+pub fn to_vec(hex: &str) -> Option<Vec<u8>> {
```

Comment:
rust-bitcoin has a helper for this I believe

Reply:
Thanks, was able to remove all that util logic.